test/e2e/login.spec.js

```diff
@@ -0,0 +1,19 @@
+import { test,expect } from "@playwright/test";
+
+test.beforeEach(async ({page})=>{
+    await page.goto('http://localhost:8080/')
+})
+
+test('should login', async ({page})=>{
+    await page.locator(`//li[@class="hide-me"]/a[@href="/account/login"]`).click()
+    await page.waitForURL('**/account/login')
+    await page.locator('//input[@id="username"]').fill('')
+    await page.locator('//input[@id="username"]').fill('openlibrary@example.com')
+    await page.locator('//input[@id="password"]').fill('')
+    await page.locator('//input[@id="password"]').fill('admin123')
+    await page.locator('//button[@title="Log In"]').click()
+    await page.waitForURL('http://localhost:8080/')
```

Comment:
Store the Url somewhere in a variable and use the variable where needed.
```suggestion
    await page.waitForURL(home_url)
```